src/components/Calculator.js

```diff
@@ -1,28 +1,49 @@
+import React, { useState } from 'react';
 import Button from './Button';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [operation, setResult] = useState({
+    total: null,
+    second: null,
+    operation: null,
+  });
+  const [displayValue, setDisplayValue] = useState('');
```

Comment:
- [ ] [OPTIONAL]
Let's improve the UX of the calculator app by ensuring that the results of a calculation is cleared just before the next calculation is done.

[calculator.webm](https://github.com/jcmunav63/math/assets/12745474/783f5a90-5772-4231-b83b-1e1efa7a8a75)